modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -6,22 +6,28 @@
 [id="granting-users-permission-to-monitor-user-defined-projects_{context}"]
 = Granting users permission to monitor user-defined projects
 
-Cluster administrators can monitor all core {product-title} and user-defined projects.
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
 
-Cluster administrators can grant developers and other users permission to monitor their own projects. Privileges are granted by assigning one of the following monitoring roles:
+You can also grant developers and other users different permissions:
 
-* The *monitoring-rules-view* cluster role provides read access to `PrometheusRule` custom resources for a project.
+* To monitor user-defined projects.
+* To configure the components that monitor user-defined projects.
+* To configure alert routing for user-defined projects.
 
-* The *monitoring-rules-edit* cluster role grants a user permission to create, modify, and delete `PrometheusRule` custom resources for a project. It also grants a user the ability to silence alerts.
+You can grant the permissions by assigning one of the following monitoring roles:
 
-* The *monitoring-edit* cluster role grants the same privileges as the `monitoring-rules-edit` cluster role. Additionally, it enables a user to create new scrape targets for services or pods. With this role, you can also create, modify, and delete `ServiceMonitor` and `PodMonitor` resources.
+|===
+|Cluster role name |Description
```

Comment:
```suggestion
|Role name |Description
```
reason, see from https://github.com/openshift/openshift-docs/pull/75852/files#r1599953870, `user-workload-monitoring-config-edit` is role, not cluster role